provider/testdata/Cargo.toml

```diff
@@ -171,6 +171,6 @@ metadata = [
     "std"
 ]
 
-fs = ["icu_provider_fs", "icu_provider_fs/deserialize_json", "std"]
-static = ["icu_provider_blob"]
+fs = ["icu_provider/deserialize_json", "icu_provider_fs", "std"]
```

Comment:
Praise: I like this idea.

Concern: I also notice that it may require child crates to add a dependency on `icu_provider` in their Cargo.toml files, whereas right now they can get everything they need from the `icu_provider_fs` crate.

I'm fine making this change for now, and if it turns out to be a problem, it's easy to add the features back to icu_provider_fs.

Reply:
So right now the FS crate only has a json and a bincode feature. If we can't keep it in sync with the data formats then we should just not have these features on provider_fs.